controllers/windowsmachine/windowsmachine_controller.go

```diff
@@ -218,6 +222,32 @@ func isWindowsMachine(labels map[string]string) bool {
 	return false
 }
 
+func isValidMachine(obj runtime.Object) bool {
+	machine := &mapi.Machine{}
+
+	// If for some reason this function called on an object which is not a Node, return
+	if kind := obj.GetObjectKind().GroupVersionKind(); kind.Kind != machine.Kind {
```

Comment:
Comment doesn't seem to match up with code, should be `not a Machine` not node, right?